lib/src/widgets/list_view_resource_widget.dart

```diff
@@ -285,10 +285,10 @@ class ListViewResourceWidget<T> extends StatelessWidget {
     switch (resource.status) {
       case Status.loading:
         if (loadingTile != null) {
-          return loadingTile!;
+          return _handleLoadingState(index, data, loadingTile!);
         }
         if (loadingTileBuilder != null) {
-          return loadingTileBuilder!();
+          return _handleLoadingState(index, data, loadingTileBuilder!());
         }
         break;
       case Status.success:
```

Comment:
In that case, will not be better to return this `_handleLoadingState` if is with `Status.loading` or `Status.success`?
Because you added the success logic on this function, so the correct name for this function should be `_mapIndexToTile`, or something like that... What you think?